### DISCUSSION TITLE
Wrong Kubernetes "route" exposed in TrueNAS Scale 2022-02

### DISCUSSION THREAD

**Discussion by bobdubois** [photoprism/photoprism#2871] [General]
When deploying Photoprism from the Official catalog in TrueNAS Scale (v. 2022-02) it comes with a default route ("web portal") that goes to address:20800/admin. However the /admin is not found and rather the correct route should be simply address:20800 (or whatever port it listens on).

**Comment by graciousgrey** [comment]
The TrueNAS Scale app is not maintained by us. I guess you need to contact their support team in this case

**Reply by bobdubois** [reply]
ok thanks for the reply!

**Reply by lastzero** [reply]
We have a site URL environment variable to specify the base URL, not sure if TrueNAS uses it. Never had an /admin route from what I know. Might have worked as the previous version didn't show error 404, but simply redirected the user to the login page.

**Comment by bobdubois** [comment]
They got it hard-coded but I provided a pull request to "/" instead of "/admin/".
Thanks for clarifying.
